requirements.txt

```diff
@@ -1,10 +1,9 @@
-
-SQLAlchemy == 1.3.23
+SQLAlchemy==1.3.23
 
 # Used for Landsat data sync
-rasterio~=1.1.8
-stactools-cli==0.1.4
-stactools-core==0.1.4
-stactools-landsat==0.1.4
-pystac==0.5.4
+# On Windows install GDAL (GDAL‑3.1.4‑cp36‑cp36m‑win_amd64.whl) and Rasterio (rasterio‑1.1.8‑cp36‑cp36m‑win_amd64.whl) from https://www.lfd.uci.edu/~gohlke/pythonlibs/
+stactools-cli==0.1.6
+stactools-core==0.1.6
+stactools-landsat==0.1.6
+pystac==0.5.6
```

Comment:
Do we need these?

Reply:
Just if we restart doing the transfer through here, but once they were already in the requirements I just updated them.
I can remove that, but the logic script is still under utils